.github/workflows/publish-packages.yml

```diff
@@ -2,20 +2,16 @@ name: Publish Hazelcast OS & EE packages
 
 on:
   push:
-    # Push to master updates the latest snapshot (HZ_VERSION taken from pom.xml)
     branches:
       - master
-    # Push of a matching tag (v*, e.g. v5.0.2) starts build with
-    # - HZ_VERSION extracted from pom.xml
-    # - PACKAGE_VERSION extracted from the tag
     tags:
       - 'v*'
```

Comment:
well, after recent changes I would say it would be better to have a single place which decides what tags are released. I mean the release pipeline is a decision point where we decide to create a tag which will result with a package release. If we add additional logic here, any future changes will require changing both places. 

In other words: let's keep packaging simple and trust that if your release pipeline creates a tag here it means we should release packages

Reply:
Understood but anyone can create a tag at any point and with any name and not only Release Pipeline
Granted that most likely it will always be by Release Pipeline but the fact that any entity can create a tag, I think best to guard against it. I am not too concerned about making future changes. If a change is needed then we make the change 

If you feel strongly about this, then feel free to resolve with a comment as I do understand where you are coming from

Reply:
> Understood but anyone can create a tag at any point and with any name and not only Release Pipeline

The same way one can create vX.Y.Z or BETA tags. AFAIR so far we haven't got a situation where somebody created a tag by accident so I would prefer to keep it simple for now and address actual problems. For now I consider a tag name as an implementation detail of release pipelines

Reply:
No worries @ldziedziul